Gemfile

```diff
@@ -53,6 +53,7 @@ gem 'bootsnap', require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'rails-controller-testing'
```

Comment:
- [ ] Kindly note that it is a strict requirement from the project requirements that you add the rspec gem in your project. Kindly ensure that you include it in the ```:development, :test``` block

![image](https://user-images.githubusercontent.com/86318284/214615469-f279bd37-b36b-4121-8aa0-8ae40504f8d7.png)

Reply:
resolved